packages/application/style/base.css

```diff
@@ -13,6 +13,13 @@
 body {
   margin: 0;
   padding: 0;
+}
+
+/*
+  Override the default background
+  See https://github.com/jupyterlab/jupyterlab/pull/16519 for more information
+*/
+.jp-ThemedContainer {
```

Comment:
I think this is because of this change. I think the right fix is to just adjust the selector to `body.jp-ThemedContainer`:

```suggestion
body.jp-ThemedContainer {
  margin: 0;
  padding: 0;
```

Reply:
This will also make embedding notebook in other websites easier (when using a full javascript rather than iframe) as the styles will not conflict :)

Reply:
Yes, tried that before but it didn't seem to be enough:

![image](https://github.com/user-attachments/assets/49299022-4ff9-48b7-9e15-31c65885169a)

Reply:
Looks like `#main.jp-ThemedContainer` also had to be defined.